GW2EIEvtcParser/EncounterLogic/Fractals/SilentSurf/Kanaxai.cs

```diff
@@ -0,0 +1,130 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using GW2EIEvtcParser.EIData;
+using GW2EIEvtcParser.Exceptions;
+using GW2EIEvtcParser.Extensions;
+using GW2EIEvtcParser.ParsedData;
+using static GW2EIEvtcParser.SkillIDs;
+using static GW2EIEvtcParser.ParserHelper;
+using static GW2EIEvtcParser.EncounterLogic.EncounterCategory;
+using static GW2EIEvtcParser.EncounterLogic.EncounterLogicUtils;
+using static GW2EIEvtcParser.EncounterLogic.EncounterLogicPhaseUtils;
+using static GW2EIEvtcParser.EncounterLogic.EncounterLogicTimeUtils;
+using static GW2EIEvtcParser.EncounterLogic.EncounterImages;
+
+namespace GW2EIEvtcParser.EncounterLogic
+{
+    internal class Kanaxai : FractalLogic
+    {
+        public Kanaxai(int triggerID) : base(triggerID)
+        {
+            MechanicList.AddRange(new List<Mechanic>
+            {
+                new PlayerDstHitMechanic(WorldCleaver, "World Cleaver", new MechanicPlotlySetting(Symbols.CircleX, Colors.Red), "WorldClv.H", "Hit by World Cleaver", "World Cleaver Hit", 150),
+                new PlayerDstBuffApplyMechanic(DreadVisage, "Dread Visage", new MechanicPlotlySetting(Symbols.DiamondOpen, Colors.Black), "DreadVis.A", "Applied Dread Visage", "Dread Visage Application", 2000),
+            });
+            Extension = "kanaxai";
+            Icon = EncounterIconKanaxai;
+            EncounterCategoryInformation.SubCategory = SubFightCategory.SilentSurf;
+            EncounterID |= EncounterIDs.FractalMasks.SilentSurfMask;
```

Comment:
add an extra "| 0x000001" there to make sure encounter ID is not 00, the 0xXXXX00 pattern is reserved for the future for map logs (well it is already in use for WvW).
Also https://github.com/baaron4/GW2-Elite-Insights-Parser/blob/master/EncounterIDs.md needs to be updated.